scripts/test_docker_image.sh

```
@@ -155,9 +179,9 @@ else
     # Start the container in the background
     docker run --name "$CONTAINER_NAME" -d -p 4000:4000 \
       -e DISCORD_BOT_TOKEN="$DISCORD_TOKEN" \
-      -e CONFIG_PATH=/app/etc/wanderer_notifier.exs \
       -e WANDERER_ENV=test \
       -e WANDERER_FEATURE_DISABLE_WEBSOCKET=true \
+      $EXTRA_ENV_VARS \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)